packages/mobile/src/screens/sign-on-screen/screens/SelectArtistScreen/selectArtistPreviewContext.tsx

```diff
@@ -0,0 +1,109 @@
+import { createContext, useCallback, useMemo, useState } from 'react'
```

Comment:
This context looks clean af 👨‍🍳 👌

Reply:
thanks! copied liberally from @amendelsohn's implementation on web